templates/terraform/decoders/disk.erb

```diff
@@ -4,6 +4,9 @@ if v, ok := res["diskEncryptionKey"]; ok {
   // The raw key won't be returned, so we need to use the original.
   transformed["rawKey"] = d.Get("disk_encryption_key.0.raw_key")
   transformed["sha256"] = original["sha256"]
+  // The response for crypto keys often includes the version of the key which needs to be removed
```

Comment:
You should add version checks in ruby here so that these don't appear in the GA version:
```
<% unless version.nil? || version == 'ga' %>
```
(you can do this now, without my changes)